### PR TITLE
Update v3.3.1

### PR DESCRIPTION
**Pull request by aaronskiba**
Fixes # Update to v3.3.1

Changes proposed in this PR:
- Merge https://github.com/portagenetwork/roadmap/commit/47be9ba2102bd834bb55b0a95c2cec38a9cc0e35
  - Conflicts existed within `_plan_text.erb` and `exportable_plan.rb`
  - **Current**, rather than **Incoming** changes were accepted to resolve conflicts in both of these files

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### [3.1.0+portage-3.1.4] - 2023-07-13
```
Comment (lagoan) [inline CHANGELOG.md]: We haven't cut the release yet so we can leave this as a part of the ```Unreleased``` section. Also, there are some details that we need to discuss about the versioning system we are using.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+
+### Changed
+
+ - Merged Release [V3.3.1](https://github.com/portagenetwork/roadmap/commit/47be9ba2102bd834bb55b0a95c2cec38a9cc0e35) into portage-deployment
```
Comment (lagoan) [inline CHANGELOG.md]: You can reference the upstream version that we merged: 3.3.1 https://github.com/DMPRoadmap/roadmap/releases/tag/v3.3.1

**Review by lagoan — CHANGES_REQUESTED** [review]
Looks good! Good job on merging to that release!

There are just a couple of minor changes to the CHANGELOG.md file that are needed. 

1. Make sure we add the information on the Unreleased portion of the document as we haven't cut a release yet.
2. Point to the release version from DMPRoadmap

Looking forward to the final result!

File: CHANGELOG.md (hunk 2 above)
Comment (aaronskiba) [reply CHANGELOG.md]: Cool, I left all of that in the Unreleased portion and replaced the reference. Let me know if anything else is required.

**Review by lagoan — APPROVED** [review]
LGTM!

**Pull request merged**